torch/_C/_distributed_c10d.pyi

```diff
@@ -1,8 +1,9 @@
 from datetime import timedelta
 from enum import Enum
-from typing import Optional, List, Any, Tuple, overload, Union
+from typing import Dict, Optional, List, Any, Tuple, overload, Union
```

Comment:
Nit: ideally we should sort it alphabetically?

Reply:
fixed